checkov/argo_workflows/runner.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+from checkov.common.output.report import CheckType
+from checkov.yaml_doc.runner import Runner as YamlRunner
+
+# Import of the checks registry for a specific resource type
+from checkov.argo_workflows.checks.registry import registry as template_registry
+
+if TYPE_CHECKING:
+    from checkov.common.checks.base_check_registry import BaseCheckRegistry
+
+
+class Runner(YamlRunner):
```

Comment:
I think it should inherit from Image reference too

Reply:
done